src/detectors/Cpp/CppLanguageDetector.ts

```diff
@@ -0,0 +1,30 @@
+import { ILanguageDetector } from '../ILanguageDetector';
+import { injectable, inject } from 'inversify';
+import { LanguageAtPath, ProgrammingLanguage } from '../../model';
+import { IFileInspector } from '../../inspectors/IFileInspector';
+import { Types } from '../../types';
+import { fileExtensionRegExp, sharedSubpath } from '../utils';
+import { uniq } from 'lodash';
+import * as nodePath from 'path';
+
+@injectable()
+export class CppLanguageDetector implements ILanguageDetector {
+  private fileInspector: IFileInspector;
+  constructor(@inject(Types.IFileInspector) fileInspector: IFileInspector) {
+    this.fileInspector = fileInspector;
+  }
+
+  async detectLanguage(): Promise<LanguageAtPath[]> {
+    const result: LanguageAtPath[] = [];
+
+    const cppFiles = await this.fileInspector.scanFor(fileExtensionRegExp(['cpp', 'hpp']), '/');
```

Comment:
The same as https://github.com/DXHeroes/dx-scanner/pull/90#discussion_r336892921

Reply:
Do you know which file can I search for ? I don't know if there's a famous package manager or something like this.

Reply:
@Sehsyha I have found this: https://stackoverflow.com/questions/27866965/does-c-have-a-package-manager-like-npm-pip-gem-etc My colleague told me, there is no unified definition file. So leave it as is. :) thanks